x-pack/plugins/secops/public/components/page/network/kpi_network/index.tsx

```diff
@@ -22,34 +23,44 @@ import * as i18n from './translations';
 
 interface KpiNetworkProps {
   data: KpiNetworkData;
+  loading: boolean;
```

Comment:
Nice! Thanks for adding the `loading` prop and `EuiLoadingSpinner` 👍 We may want to abstract this into a custom HoC here soon to provide loading for widgets like these KPI's and the Summary components (similar to what `load_more_table` does for all the tables).